onnxruntime/test/providers/cpu/controlflow/scan_test.cc

```diff
@@ -415,6 +415,8 @@ static void RunTest_v9(const std::string test_name, int64_t sequence_len, int64_
 
     test.Run(expect_result, failure_message, options.excluded_provider_types, nullptr, &execution_providers);
   } else {
+    std::unique_ptr<onnxruntime::Model> scan_model = test.BuildGraph();
+    onnxruntime::Model::Save(*scan_model, "/bert_ort/liqun/test_out/scan_model.onnx");
```

Comment:
I guess it's for debugging the regression, please remember to remove before merge.